frontend/ICard/.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   root: true,
   extends: '@react-native-community',
+  rules: {'prettier/prettier': ['error', {endOfLine: 'auto'}]},
```

Comment:
I see prettier is working nicely, but I don’t see it installed in package.json. Will this work on a machine without prettier’s extension installed? (I have the VS Code extension in my machine)

Reply:
If installing prettier, consider adding eslint-plugin-prettier to make prettier works nicely with eslint

Reply:
I haven't test without prettier extension, I will include prettier inside package.json. Will further include eslint-plugin-prettier to the installation too

Reply:
Fixed with https://github.com/UofA-Blueprint/ICard/pull/1/commits/138a50024e8e2a778eb948c65f80199aadbfb309